src/main/java/dev/snowdrop/jira/atlassian/model/Release.java

```diff
@@ -1,50 +1,28 @@
 package dev.snowdrop.jira.atlassian.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 import java.util.List;
 
 public class Release {
-	private String jiraProject;
-	private String jiraKey;
-	private String version;
-	private String longVersionName;
```

Comment:
We still need this long title description as it is used to create the JIRA component issue